plugins/catalog-react/api-report.md

```diff
@@ -742,6 +761,27 @@ export function reduceEntityFilters(
 // @public (undocumented)
 export const rootRoute: RouteRef<undefined>;
 
+// @public
+export interface StarredEntitiesApi {
+  star(entity: Entity): Promise<void>;
+  starredEntities$(): Observable<StarredEntitiesApiObservable>;
+  toggleStarred(entity: Entity): Promise<void>;
+  unstar(entity: Entity): Promise<void>;
```

Comment:
is it possible to remove `star` and `unstar` and just use `toggleStarred`, feels like there's at least one extra method here that's not needed